modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/tracing/opentelemetry/management/TelemetryConstants.java

```diff
@@ -55,6 +55,12 @@ public class TelemetryConstants {
     public static final String DEFAULT_ZIPKIN_HOST = "localhost";
     public static final String DEFAULT_ZIPKIN_PORT = "9411";
     public static final String ZIPKIN_API_CONTEXT = "/api/v2/spans";
+    static final String LATENCY = "Latency";
+    static final String OPERATION_NAME = "Operation";
```

Comment:
Let's rename `OPERATION_NAME` as `SPAN_NAME = "Span Name";` as discussed in the earlier comment.

Reply:
Noted and resolved.